src/App.js

```diff
@@ -1,15 +1,27 @@
 import React from "react";
-
 import Bookings from "./Bookings";
 import "./App.css";
+import Heading from "./components/Heading";
+import TouristInfoCards from "./components/TouristInfoCards";
+import Footer from "./components/Footer";
+import Restaurant from "./components/Restaurant";
 
 const App = () => {
+  const addresses = [
```

Comment:
This variable makes it seem like the list contains only addresses. However it has an email and a phone number. It might be better to rename it to something like details.